tardis/simulation/base.py

```diff
@@ -272,6 +275,11 @@ def advance_state(self, emitted_luminosity):
         -------
             converged : bool
         """
+        current_radiation_field_state = DilutePlanckianRadiationField(
+            self.simulation_state.t_radiative,
```

Comment:
Please use the `simulation_state.radiation_field_state.*` properties.  These are passed to the plasma which acts on the entire base geometry-sized arrays. Adding a slice or active method to the DilutePlanckianRadiationField would be preferable if a user is trying to only access the exposed information

Reply:
Okay, I couldn't find the radiation_field_state on the simulation_state when I looked (not very carefully) so I can easily use that instead.

Reply:
It is passed as an argument to the SimulationState constructor and the t_radiative and dilution_factor properties grab it from it's radiation_field_state attribute.